tutorials-v5/miscellaneous/excitation-number-restricted-states-jc-chain.md

```diff
@@ -0,0 +1,264 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+<!-- #region -->
+# Excitation-number-restricted states: Jaynes-Cummings Chain
+
+Authors: Robert Johansson (jrjohansson@gmail.com), Neill Lambert (nwlambert@gmail.com), Maximilian Meyer-Mölleringhof (m.meyermoelleringhof@gmail.com)
+
+## Introduction
+
+The ENR functions construct a basis set for multipartite systems which contains only states that have an overall number of excitations.
+This is particularly useful for systems where the model conserves excitation number, as in the JC-chain example below.
+
+We can see this by considering a system consisting of 4 modes, each with 5 states.
+The total hilbert space size is $5^4 = 625$.
+If we are only interested in states that contain up to 2 excitations, we only need to include states such as
+
+
+    (0, 0, 0, 0)
+    (0, 0, 0, 1)
+    (0, 0, 0, 2)
+    (0, 0, 1, 0)
+    (0, 0, 1, 1)
+    (0, 0, 2, 0)
+    ...
+
+The ENR fucntions create operators and states for the 4 modes that act within this state space.
+For example,
+
+```python
+a1, a2, a3, a4 = enr_destroy([5, 5, 5, 5], excitations=2)
+```
+
+creates destruction operators for each mode.
+From this point onwards, the annihiltion operators a1, ..., a4 can be used to setup a Hamiltonian, collapse operators and expectation-value operators, etc., following the usual patterne.
+
+In this example we outline the advantage of ENR states by comparing them with the regular qutip implementation.
+For this we calculate the time evolution and the partial trace for each and see consistent results with notable performance improvements.
+
+#### Be aware!
+
+Many default functions in QuTiP will fail on states and operators constructed with this method.
+Additionally, using this formalism, annihilation and creation operators of different sub-systems no longer commute.
+Therefore, when constructing Hamiltonians, annihilation operators must be on the right and creation operators on the left (see the offical publication for QuTiP v5 for more info). 
```

Comment:
perhaps below we can explicitly show how this ordering gives rise to two different operators?

Reply:
How explicit do you mean?

Reply:
hmm i guess just showing that in the in the full case d[0].dag() * d[1]==d[1] * d[0].dag()  but in the ENR case its fails, even just with an assert?